### ISSUE TITLE
Cannot reach actions with keyboard

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#23379]
Testing #23287

In the new keybindings experience.

Cannot reach the edit/remove/add actions on a row with the keyboard. I have tried arrow left/right or tab, but I could not make focus reach those actions.

**Comment by ramya-rao-a** [comment]
Edit action can be reached using the shortcut `Ctrl+K Ctrl+K`
But it would be more obvious if edit gets triggered on Enter or Space.

As for reaching rest of the actions, I propose #23496

**Comment by sandy081** [comment]
While navigating the keybindings using arrows and when the row is focused / selected you can navigate to edit action by pressing `tab`. 

You can reach other actions by triggering the context menu using the platform specific keyboard shortcuts (Shift+f10 for windows)
